### PR TITLE
Added user stories

### PR DESCRIPTION
**Pull request by Sisyphus25**
- Add more user stories to cover the targeted users

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |edit the detail of a person
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: details of a person

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+ |easily make change to their detail when they updated their contact
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: easily make changes to their details when they update their contact

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* * *` |user |delete a person |remove entries that I no longer need
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher| delete the contact of a student | remove contacts that I no longer need

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* * *` |user |create a class |group and manage students who are taking the same class
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher | create a class|

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* * *` |user |add persons to a class |
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher | add students to a class| group them for easy perusal

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* * *` |user |see all persons in a class |see the list of all students taking a class
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher | list all the students in a particular class | know all the students taking that class

File: docs/DeveloperGuide.adoc [hunk 7]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |add appointment with a person to my schedule |be reminded of the appointment
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher | add appointments with a student to my schedule| be reminded of the appointment

File: docs/DeveloperGuide.adoc [hunk 8]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
+|`* *` |user |delete appointment from my schedule |remove appointment I no longer need to be reminded
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher| delete appointments from my schedule| clear appointments I no longer need to be reminded about

File: docs/DeveloperGuide.adoc [hunk 9]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |user with many persons in address book |tag a person |mark their contact with a detail
```
Comment (shanmu9898) [inline docs/DeveloperGuide.adoc]: teacher with many students in address book | tag a student | mark their contact with details

Comment (randypx) [reply docs/DeveloperGuide.adoc]: no need for 'with many students in address book'. User should be able to tag regardless of number of contacts

File: docs/DeveloperGuide.adoc (hunk 8 above)
Comment (randypx) [reply docs/DeveloperGuide.adoc]: add a list all appointments as well

File: docs/DeveloperGuide.adoc [hunk 10]
```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
```
Comment (randypx) [inline docs/DeveloperGuide.adoc]: There is a delete a person user story above as well. Please remove either one.

**Pull request merged**